### PR TITLE
Unskip tests

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-ci run all integration tests

**Comment by janisz** [comment]
@mesosphere-ci run linux integration tests

**Comment by janisz** [comment]
@mesosphere-ci run Windows integration tests

**Comment by janisz** [comment]
@mesosphere-ci run mac integration tests

**Comment by janisz** [comment]
@mesosphere-ci run Windows integration tests

**Comment by janisz** [comment]
@mesosphere-ci run Windows integration tests

**Comment by janisz** [comment]
@mesosphere-ci run all integration tests

File: python/lib/dcoscli/tests/integrations/test_package.py [hunk 1]
```diff
@@ -312,22 +312,20 @@ def test_uninstall_cli_only_when_no_apps_remain():
...
+@pytest.mark.parametrize("command_to_run,expected_error", [
+    ('helloworld --yes --options=asdf.json', b"Error: couldn't find options file 'asdf.json'\n"),
+    ('helloworld --app --cli', b'Error: --app and --cli are mutually exclusive\n'),
+    ('cassandra --package-version=a.b.c', b'Error: Version [a.b.c] of package [cassandra] not found\n'),
+    ('missing-package', b'Error: Package [missing-package] not found\n'),
+])
+def test_install_error(command_to_run, expected_error):
+    returncode, stdout, stderr = exec_command(
+        ['dcos', 'package', 'install'] + command_to_run.split(' '),
+    )
```
Comment (drewvanstone) [inline python/lib/dcoscli/tests/integrations/test_package.py:326]: Nice!!

**Review by drewvanstone — COMMENTED** [review]
Is this still WIP or is it ready to go?

**Comment by janisz** [comment]
@drewvanstone It's blocked by https://github.com/mesosphere/universe/pull/2424

**Comment by janisz** [comment]
@mesosphere-ci run all integration tests

**Comment by janisz** [comment]
@mesosphere-ci run all integration tests

**Comment by janisz** [comment]
:wave: @drewvanstone @br-lewis @makkes @bamarni @rgo3  Can you review, please?

**Review by br-lewis — APPROVED** [review]

**Review by makkes — APPROVED** [review]

**Comment by janisz** [comment]
@mesosphere-ci run linux integration tests

**Comment by janisz** [comment]
@mesosphere-ci run linux integration tests

**Pull request merged**